webdriver-extension/index.bs

```diff
@@ -0,0 +1,532 @@
+supported bounds are freely defined by user agent.
+
+<div class='note'>
+Note: The [=mock sensor=] defined in this specification is not intended be used by non-testing-related web content.
+The UA MAY choose to expose [=mock sensor=] interface only when a runtime or compile-time flag has been set.
+</div>
+
+## MockSensorConfiguration dictionary ## {#dictionary-mocksensorconfiguration}
+
+<pre class='idl'>
+  dictionary MockSensorConfiguration {
+    required MockSensorType mockSensorType;
+    boolean sensorShouldConnected = true;
+    double? maxSamplingFrequency;
+    double? minSamplingFrequency;
+  };
+</pre>
+
+The {{MockSensorConfiguration}} dictionary is used to [[#create-mock-sensor-command|create mock sensor]].
+
+: {{MockSensorConfiguration/mockSensorType}} member
+:: A {{MockSensorType}} that is used to set <a>mock sensor type</a>.
+
+: {{MockSensorConfiguration/sensorShouldConnected}} member
+:: A boolean. When set to false the user agent should intecept the result of invoking <a>connect to sensor</a>
+   with associated <a>Sensor</a> instance as argument to false, otherwise true.
+
+: {{MockSensorConfiguration/maxSamplingFrequency}} member
+:: A double representing frequency in Hz that is used to set maximum supported <a>sampling frequency</a> for the associated [=mock sensor=].
+
+: {{MockSensorConfiguration/minSamplingFrequency}} member
+:: A double representing frequency in Hz that is used to set minimum supported <a>sampling frequency</a> for the associated [=mock sensor=].
+
+## MockSensor dictionary ## {#dictionary-mocksensor}
+<pre class='idl'>
+  dictionary MockSensor {
+    double maxSamplingFrequency;
+    double minSamplingFrequency;
+    double requestedSamplingFrequency;
+  };
+</pre>
+The {{MockSensor}} dictionary provides information of a <a>mock sensor</a>.
+
+: {{MockSensor/maxSamplingFrequency}} member
+:: A double representing frequency in Hz that indicates the maximum supported <a>sampling frequency</a> of the associated [=mock sensor=].
+
+: {{MockSensor/minSamplingFrequency}} member
+:: A double representing frequency in Hz that indicates the minimum supported <a>sampling frequency</a> of the associated [=mock sensor=].
+
+: {{MockSensor/requestedSamplingFrequency}} member
+:: A double representing frequency in Hz that indicates the <a>requested sampling frequency</a> of the associated [=mock sensor=].
+
+A <dfn>serialized mock sensor</dfn> is a JSON <a>Object</a> where a [=mock sensor=]'s fields listed in the {{MockSensor}} dictionary are mapped
+using the |JSON Key| and the associated field’s value from the available [=mock sensor=] in <a>current browsing context</a>.
+
+## Mock sensor type ## {#section-mock-sensor-type}
+
+A <dfn id="mock-sensor-type">mock sensor type</dfn> is equivalent to a <a>sensor type</a>’s associated {{Sensor}} subclass.
+
+<pre class='idl'>
+  enum MockSensorType {
+    "ambient-light",
```

Comment:
Is there a list in another spec that this should stay in sync with? Cross-linking the two lists like in source code would work :)